commands/hugo.go

```diff
@@ -222,7 +222,7 @@ func initHugoBuildCommonFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&contentDir, "contentDir", "c", "", "filesystem path to content directory")
 	cmd.Flags().StringVarP(&layoutDir, "layoutDir", "l", "", "filesystem path to layout directory")
 	cmd.Flags().StringVarP(&cacheDir, "cacheDir", "", "", "filesystem path to cache directory. Defaults: $TMPDIR/hugo_cache/")
-	cmd.Flags().BoolVarP(&ignoreCache, "ignoreCache", "", false, "Ignores the cache directory for reading but still writes to it")
```

Comment:
All below /docs/commands are autogenerated (we typically do it before a new releasee), so -- although this doesn't hurt it makes the changes harder to read, so revert it.